src/package.json

```diff
@@ -20,6 +20,8 @@
         "plugin.tscwatch": "npm run tsc -- -w",
         "demo.ios": "npm i && npm run tsc && cd ../demo && tns run ios --syncAllFiles",
         "demo.android": "npm i && npm run tsc && cd ../demo && tns run android --syncAllFiles",
+        "demo.reset": "cd ../demo && rm -rf platforms && cd ../src",
+        "plugin.prepare": "npm run tsc && cd ../demo && tns plugin remove nativescript-yourplugin && tns plugin add ../src && cd ../src",
```

Comment:
When do you use `plugin.prepare`? 

Let me give some more insights to understand why I'm asking: The development flow that we are promoting and use internally is to use `npm link` since it's much faster for development than using `tns plugin add/remove` commands. The only case when you would like to use the commands is when you add some platform specific configurations in your plugin that you want to apply. Since this is not something you do frequently during your development, that's why we decided the user can just use the raw commands`tns plugin add\remove` instead of having explicit npm command. Also running `plugin.prepare` will lose the link to the plugin source. Let's discuss the cases this command is needed and decide if it should be in the seed. In case we leave it, we need to add `npm run plugin.link` at the end to link to the plugin to the plugin source.

And some more general thoughts....We try to keep the seed simple in order to prevent having files and commands that are used for specific types of plugins. That's why we cleaned up some of the commands from the original code of the seed. What we strive for is to recommend the minimum needed to develop your plugin.